src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
+      <li>
```

Comment:
It is also fashionable to create a PostsListItem component that will display the Post entity and make a map for it here

like this

```javascript
    {posts.map((post) => <PostsListItem title={post.title} body={post.body}  /> )} // ore you can use  <PostsListItem {...post}  />
)
    
``` 
PostsListItem:
```javascript
 const PostsListItem: React.FC<Props> = ({ title, body }) => (
    <li>
        <b>{title}</b>
        <div>{body}</div>
         <hr />
    </li>
```